### PR TITLE
Removed lesser used chat modules

### PR DESCRIPTION
**Pull request by czho**
**Describe the pull**
Removed lesser used chat modules in favor of the chat plugin: chat-plus

**Describe how this pull is helpful**
Clears up confusion and removes bloat.

**Additional context**
https://github.com/lambda-plugins/ChatPlus

### CODE REVIEW

File: src/main/kotlin/com/lambda/client/module/modules/chat/CustomChat.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ internal object CustomChat : Module(
 
     private val modifier = newMessageModifier(
         filter = {
-            (commands || MessageDetection.Command.ANY detectNot it.packet.message)
+            (commands)
```
Comment (5HT2) [inline src/main/kotlin/com/lambda/client/module/modules/chat/CustomChat.kt]: This breaks CustomChat

File: src/main/kotlin/com/lambda/client/module/modules/chat/FancyChat.kt [hunk 2]
```diff
@@ -25,7 +25,7 @@ internal object FancyChat : Module(
 
     private val modifier = newMessageModifier(
         filter = {
-            (commands.value || MessageDetection.Command.ANY detectNot it.packet.message)
+            (commands.value)
```
Comment (5HT2) [inline src/main/kotlin/com/lambda/client/module/modules/chat/FancyChat.kt]: This breaks FancyChat

File: src/main/kotlin/com/lambda/client/util/text/MessageDetection.kt [hunk 3]
```diff
@@ -18,10 +17,6 @@ object MessageDetection {
         ANY_EXCEPT_DELIMITER {
```
Comment (5HT2) [inline src/main/kotlin/com/lambda/client/util/text/MessageDetection.kt]: This should be renamed to ANY

File: src/main/kotlin/com/lambda/client/module/modules/chat/FancyChat.kt (hunk 2 above)
Comment (czho) [reply src/main/kotlin/com/lambda/client/module/modules/chat/FancyChat.kt]: oh shit ur right, I must have misunder stood the code when i was applying the fix

File: src/main/kotlin/com/lambda/client/util/text/MessageDetection.kt (hunk 3 above)
Comment (czho) [reply src/main/kotlin/com/lambda/client/util/text/MessageDetection.kt]: got it, then I can fix fancy chat

**Review by Avanatiker — APPROVED** [review]

**Pull request merged**